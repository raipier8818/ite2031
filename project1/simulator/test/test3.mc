8519693
8585230
8650767
1310722
16908294
131077
196609
720899
2424837
17170425
16842748
12779536
25165824
6
1
-1
0
