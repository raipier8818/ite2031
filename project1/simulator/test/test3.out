memory[0]=8519693
memory[1]=8585230
memory[2]=8650767
memory[3]=1310722
memory[4]=16908294
memory[5]=131077
memory[6]=196609
memory[7]=720899
memory[8]=2424837
memory[9]=17170425
memory[10]=16842748
memory[11]=12779536
memory[12]=25165824
memory[13]=6
memory[14]=1
memory[15]=-1
memory[16]=0

@@@
state:
	pc 0
	memory:
		mem[ 0 ] 8519693
		mem[ 1 ] 8585230
		mem[ 2 ] 8650767
		mem[ 3 ] 1310722
		mem[ 4 ] 16908294
		mem[ 5 ] 131077
		mem[ 6 ] 196609
		mem[ 7 ] 720899
		mem[ 8 ] 2424837
		mem[ 9 ] 17170425
		mem[ 10 ] 16842748
		mem[ 11 ] 12779536
		mem[ 12 ] 25165824
		mem[ 13 ] 6
		mem[ 14 ] 1
		mem[ 15 ] -1
		mem[ 16 ] 0
	registers:
		reg[ 0 ] 0
		reg[ 1 ] 0
		reg[ 2 ] 0
		reg[ 3 ] 0
		reg[ 4 ] 0
		reg[ 5 ] 0
		reg[ 6 ] 0
		reg[ 7 ] 0
end state

@@@
state:
	pc 1
	memory:
		mem[ 0 ] 8519693
		mem[ 1 ] 8585230
		mem[ 2 ] 8650767
		mem[ 3 ] 1310722
		mem[ 4 ] 16908294
		mem[ 5 ] 131077
		mem[ 6 ] 196609
		mem[ 7 ] 720899
		mem[ 8 ] 2424837
		mem[ 9 ] 17170425
		mem[ 10 ] 16842748
		mem[ 11 ] 12779536
		mem[ 12 ] 25165824
		mem[ 13 ] 6
		mem[ 14 ] 1
		mem[ 15 ] -1
		mem[ 16 ] 0
	registers:
		reg[ 0 ] 0
		reg[ 1 ] 0
		reg[ 2 ] 6
		reg[ 3 ] 0
		reg[ 4 ] 0
		reg[ 5 ] 0
		reg[ 6 ] 0
		reg[ 7 ] 0
end state

@@@
state:
	pc 2
	memory:
		mem[ 0 ] 8519693
		mem[ 1 ] 8585230
		mem[ 2 ] 8650767
		mem[ 3 ] 1310722
		mem[ 4 ] 16908294
		mem[ 5 ] 131077
		mem[ 6 ] 196609
		mem[ 7 ] 720899
		mem[ 8 ] 2424837
		mem[ 9 ] 17170425
		mem[ 10 ] 16842748
		mem[ 11 ] 12779536
		mem[ 12 ] 25165824
		mem[ 13 ] 6
		mem[ 14 ] 1
		mem[ 15 ] -1
		mem[ 16 ] 0
	registers:
		reg[ 0 ] 0
		reg[ 1 ] 0
		reg[ 2 ] 6
		reg[ 3 ] 1
		reg[ 4 ] 0
		reg[ 5 ] 0
		reg[ 6 ] 0
		reg[ 7 ] 0
end state

@@@
state:
	pc 3
	memory:
		mem[ 0 ] 8519693
		mem[ 1 ] 8585230
		mem[ 2 ] 8650767
		mem[ 3 ] 1310722
		mem[ 4 ] 16908294
		mem[ 5 ] 131077
		mem[ 6 ] 196609
		mem[ 7 ] 720899
		mem[ 8 ] 2424837
		mem[ 9 ] 17170425
		mem[ 10 ] 16842748
		mem[ 11 ] 12779536
		mem[ 12 ] 25165824
		mem[ 13 ] 6
		mem[ 14 ] 1
		mem[ 15 ] -1
		mem[ 16 ] 0
	registers:
		reg[ 0 ] 0
		reg[ 1 ] 0
		reg[ 2 ] 6
		reg[ 3 ] 1
		reg[ 4 ] -1
		reg[ 5 ] 0
		reg[ 6 ] 0
		reg[ 7 ] 0
end state

@@@
state:
	pc 4
	memory:
		mem[ 0 ] 8519693
		mem[ 1 ] 8585230
		mem[ 2 ] 8650767
		mem[ 3 ] 1310722
		mem[ 4 ] 16908294
		mem[ 5 ] 131077
		mem[ 6 ] 196609
		mem[ 7 ] 720899
		mem[ 8 ] 2424837
		mem[ 9 ] 17170425
		mem[ 10 ] 16842748
		mem[ 11 ] 12779536
		mem[ 12 ] 25165824
		mem[ 13 ] 6
		mem[ 14 ] 1
		mem[ 15 ] -1
		mem[ 16 ] 0
	registers:
		reg[ 0 ] 0
		reg[ 1 ] 0
		reg[ 2 ] 5
		reg[ 3 ] 1
		reg[ 4 ] -1
		reg[ 5 ] 0
		reg[ 6 ] 0
		reg[ 7 ] 0
end state

@@@
state:
	pc 5
	memory:
		mem[ 0 ] 8519693
		mem[ 1 ] 8585230
		mem[ 2 ] 8650767
		mem[ 3 ] 1310722
		mem[ 4 ] 16908294
		mem[ 5 ] 131077
		mem[ 6 ] 196609
		mem[ 7 ] 720899
		mem[ 8 ] 2424837
		mem[ 9 ] 17170425
		mem[ 10 ] 16842748
		mem[ 11 ] 12779536
		mem[ 12 ] 25165824
		mem[ 13 ] 6
		mem[ 14 ] 1
		mem[ 15 ] -1
		mem[ 16 ] 0
	registers:
		reg[ 0 ] 0
		reg[ 1 ] 0
		reg[ 2 ] 5
		reg[ 3 ] 1
		reg[ 4 ] -1
		reg[ 5 ] 0
		reg[ 6 ] 0
		reg[ 7 ] 0
end state

@@@
state:
	pc 6
	memory:
		mem[ 0 ] 8519693
		mem[ 1 ] 8585230
		mem[ 2 ] 8650767
		mem[ 3 ] 1310722
		mem[ 4 ] 16908294
		mem[ 5 ] 131077
		mem[ 6 ] 196609
		mem[ 7 ] 720899
		mem[ 8 ] 2424837
		mem[ 9 ] 17170425
		mem[ 10 ] 16842748
		mem[ 11 ] 12779536
		mem[ 12 ] 25165824
		mem[ 13 ] 6
		mem[ 14 ] 1
		mem[ 15 ] -1
		mem[ 16 ] 0
	registers:
		reg[ 0 ] 0
		reg[ 1 ] 0
		reg[ 2 ] 5
		reg[ 3 ] 1
		reg[ 4 ] -1
		reg[ 5 ] 5
		reg[ 6 ] 0
		reg[ 7 ] 0
end state

@@@
state:
	pc 7
	memory:
		mem[ 0 ] 8519693
		mem[ 1 ] 8585230
		mem[ 2 ] 8650767
		mem[ 3 ] 1310722
		mem[ 4 ] 16908294
		mem[ 5 ] 131077
		mem[ 6 ] 196609
		mem[ 7 ] 720899
		mem[ 8 ] 2424837
		mem[ 9 ] 17170425
		mem[ 10 ] 16842748
		mem[ 11 ] 12779536
		mem[ 12 ] 25165824
		mem[ 13 ] 6
		mem[ 14 ] 1
		mem[ 15 ] -1
		mem[ 16 ] 0
	registers:
		reg[ 0 ] 0
		reg[ 1 ] 1
		reg[ 2 ] 5
		reg[ 3 ] 1
		reg[ 4 ] -1
		reg[ 5 ] 5
		reg[ 6 ] 0
		reg[ 7 ] 0
end state

@@@
state:
	pc 8
	memory:
		mem[ 0 ] 8519693
		mem[ 1 ] 8585230
		mem[ 2 ] 8650767
		mem[ 3 ] 1310722
		mem[ 4 ] 16908294
		mem[ 5 ] 131077
		mem[ 6 ] 196609
		mem[ 7 ] 720899
		mem[ 8 ] 2424837
		mem[ 9 ] 17170425
		mem[ 10 ] 16842748
		mem[ 11 ] 12779536
		mem[ 12 ] 25165824
		mem[ 13 ] 6
		mem[ 14 ] 1
		mem[ 15 ] -1
		mem[ 16 ] 0
	registers:
		reg[ 0 ] 0
		reg[ 1 ] 1
		reg[ 2 ] 5
		reg[ 3 ] 2
		reg[ 4 ] -1
		reg[ 5 ] 5
		reg[ 6 ] 0
		reg[ 7 ] 0
end state

@@@
state:
	pc 9
	memory:
		mem[ 0 ] 8519693
		mem[ 1 ] 8585230
		mem[ 2 ] 8650767
		mem[ 3 ] 1310722
		mem[ 4 ] 16908294
		mem[ 5 ] 131077
		mem[ 6 ] 196609
		mem[ 7 ] 720899
		mem[ 8 ] 2424837
		mem[ 9 ] 17170425
		mem[ 10 ] 16842748
		mem[ 11 ] 12779536
		mem[ 12 ] 25165824
		mem[ 13 ] 6
		mem[ 14 ] 1
		mem[ 15 ] -1
		mem[ 16 ] 0
	registers:
		reg[ 0 ] 0
		reg[ 1 ] 1
		reg[ 2 ] 5
		reg[ 3 ] 2
		reg[ 4 ] -1
		reg[ 5 ] 4
		reg[ 6 ] 0
		reg[ 7 ] 0
end state
Invalid address
